File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/OrderStatisticsVO.kt
package com.sdkwork.app

data class OrderStatisticsVO(
    val createdAt: String? = null,
    val updatedAt: String? = null,
    val totalOrders: Int? = null,
    val pendingPayment: Int? = null,
    val pendingShipment: Int? = null,
    val pendingReceipt: Int? = null,
    val completed: Int? = null,
    val totalAmount: String? = null
)

File: sdkwork-app-sdk-typescript/src/types/order-statistics-vo.ts
/** 订单统计响应 */
export interface OrderStatisticsVO {
  /** 创建时间 */
  createdAt?: string;
  /** 更新时间 */
  updatedAt?: string;
  /** 总订单数 */
  totalOrders?: number;
  /** 待支付订单数 */
  pendingPayment?: number;
  /** 待发货订单数 */
  pendingShipment?: number;
  /** 待收货订单数 */
  pendingReceipt?: number;
  /** 已完成订单数 */
  completed?: number;
  /** 总交易金额 */
  totalAmount?: string;
}
